CorsixTH/Lua/hospital.lua

```diff
@@ -1112,9 +1113,10 @@ function Hospital:createEmergency(emergency)
       },
     }
     self.world.ui.bottom_panel:queueMessage("emergency", message, nil, Date.hoursPerDay() * 16, 2) -- automatically refuse after 16 days
-    outcome = true
+    return err -- sucessfully created
   end
-  return outcome
+  err = "no_helipad"
+  return err
```

Comment:
There is actually no reason to have an `err` variable in this function. You can simply return the value directly, For the line that I marked, it would be `return "no_helipad"` (but you likely already guessed that).